src/test/java/com/amplitude/HttpTransportTest.java

```diff
@@ -94,9 +100,9 @@ public void onLogEventServerResponse(Event event, int status, String message) {
     httpTransport.setHttpCall(httpCall);
     httpTransport.setCallbacks(callbacks);
     httpTransport.retryEvents(events, invalidResponse);
-    assertTrue(latch.await(1L, TimeUnit.SECONDS));
+    assertTrue(latch.await(1000000L, TimeUnit.SECONDS));
```

Comment:
do we need the large number like this?

Reply:
Whoops, thanks for catching this, I was testing :)